doc/fundamentals/append_overwrite.md

```diff
@@ -0,0 +1,616 @@
+
+When you have loaded data via a `--split` flag, as discussed above, you can use the glob to select the split-files to be sent to a pipeline.  A detailed discussion of this is available in the Pachyderm cookbook section [Splitting Data for Distributed Processing](http://docs.pachyderm.io/en/latest/cookbook/splitting.html#splitting-data-for-distributed-processing).
+
+### Lifecycle of a datum
+
+There are four basic rules to how Pachyderm will process your data in the pipelines you create.
+
+- Pachyderm will split your input into individual datums _[as you specified](./reference/pipeline_spec.html#the-input-glob-pattern)_ in your pipeline spec
+- each datum will be processed independently, using _[the parallelism you specified](../reference/pipeline_spec.html#parallelism-spec-optional)_ in your pipeline spec, with no guarantee of the order of processing
+- Pachyderm will merge the output of each pod into each output file, with no guarantee of ordering
+- the output files will look as if everything was done in one processing step
+
+If one of your pipelines is written to take all its input, whatever it may be, process it, and put the output into one file, the final output will be one file.  The many datums in your pipeline's input would become one file at the end, with the processing of every datum reflected in that file.
+
+If you write a pipeline to write to multiple files, each of those files may contained merged data that looks as if it were all processed in one step, even if you would have specified the pipeline to process each datum in parallel.
+
+The Pachyderm Pipeline System works with the Pachyderm File System to make sure that files output by each pipeline are merged successfully at the end of every commit.
+
+#### Cross and union inputs
+
+When creating pipelines, you can use "union" and "cross" operations to combine inputs.
+
+[Union input](http://docs.pachyderm.io/en/latest/reference/pipeline_spec.html#union-input) will combine each of the datums in the input repos as one set of datums. The result is that the number of datums processed is the sum of all the datums in each repo.
+
+<img alt="Union input animation: shows how each datum is comprised of each datum in every input to the union." src="../_images/union.gif" width="200px" height="200px"/>
+
+For example, let's say you have two input repos, A and B.  Each of them contain three files with the same names: my-data-file-1.txt, my-data-file-2.txt, and my-data-file-3.txt, but different content.  If you were to cross them in a pipeline, the "input" object in the pipeline spec might look like this.
```

Comment:
This looks like a bug in the syntax highlighter in github.  The directory "_images" starts with a _, which the simplistic highlighter assumes was markup rather than part of the content.  Sigh.  Fixed by the conversion to rst, because it doesn't require me to embed html to resize an image.